lib/run-status.js

```diff
@@ -194,6 +194,18 @@ class RunStatus extends Emittery {
 			this.pendingTests.get(event.testFile).delete(event.title);
 		}
 	}
+
+	getFailedTestFiles() {
+		const result = [];
+		const iterator = this.stats.byFile.keys();
+		for (let {value, done} = iterator.next(); !done; {value, done} = iterator.next()) {
+			if (this.stats.byFile.get(value).failedTests) {
+				result.push(value);
+			}
+		}
+
+		return result;
```

Comment:
You can use a `for` / `of` loop when handling iterators. However this can be dealt with much more succinctly:

```suggestion
		return [...this.stats.byFile.values()].filter({ failedTests }) => failedTests)
```